src_files.yml

```diff
@@ -1,6 +1,10 @@
 axi:
+  vlog_opts: [
+    -L common_cells_lib
+  ]
   incdirs:
     - include
+    - ../../common_cells/include
```

Comment:
Is the common convention of including directories from other dependencies in ipapprox two levels up (`../..`)? Or will this only work for a specific project?

Reply:
Normally, you nest axi related ips into a top level axi directory. Common_cells is then on the same level as that top level axi directory. This is at least done in the projects that (still) use ipapprox (pulp, pulpissimo and our internal forks thereof).

Reply:
Okay, then I am fine with it.